sdk/core/Azure.Core.TestFramework/src/testproxy.json

```diff
@@ -488,6 +488,58 @@
           }
         }
       }
+    },
+    "/admin/addtransform?overload=HeaderTransform": {
```

Comment:
just curious - why are the test proxy client types generated here rather than consumed as a nuget package?

Reply:
Well ultimately the swagger should probably live in the tools repo so that all languages can generate it. But other languages probably wouldn't use a .NET package so there is no reason to split it out from the Test Framework project.

Reply:
We should consider filing an issue to have this done via the Proxy's build pipeline and release it as a language specific package.

Reply:
See https://github.com/Azure/azure-sdk-tools/issues/2400